gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -264,9 +265,9 @@ public void orchestrate(Spec spec, Properties jobProps, long triggerTimestampMil
         String flowExecutionId = flowMetadata.get(TimingEvent.FlowEventConstants.FLOW_EXECUTION_ID_FIELD);
         DagActionStore.DagAction flowAction =
             new DagActionStore.DagAction(flowGroup, flowName, flowExecutionId, DagActionStore.FlowActionType.LAUNCH);
-        flowTriggerHandler.get().handleTriggerEvent(jobProps, flowAction, triggerTimestampMillis);
-        _log.info("Multi-active scheduler finished handling trigger event: [{}, triggerEventTimestamp: {}]", flowAction,
-            triggerTimestampMillis);
+        flowTriggerHandler.get().handleTriggerEvent(jobProps, flowAction, triggerTimestampMillis, isReminderEvent);
+        _log.info("Multi-active scheduler finished handling trigger event: [{}, triggerEventTimestamp: {}]" +
+            "(is " + (isReminderEvent ? "reminder" : "original") + ")", flowAction, triggerTimestampMillis);
```

Comment:
now that you're using the `{}` placeholders, it seems clearest to use throughout, as:
```
log.info("Multi-active ... event: [{}, triggerEventTimestamp: {}] (is {})", flowAction, ttMillis, isReminderEvent ? "reminder" : "original");
```
writing it that way makes spotting the lack of space before `"(is "` easier to see.

also, to entertain which we prefer:
```
event: [{1}, triggerEventTimestamp: {2}] (is {3})
{3} event: [{1}, triggerEventTimestamp: {2}]
event: [{1}, is: {3}, triggerEventTimestamp: {2}]
```
(numbered purely for illustration)